README.md

```diff
@@ -71,6 +71,22 @@ Supported Tor Releases
 -----------------------
 - tor >= 0.3.5.x
 
+Example Playbook
+--------------
+Variables are set in playbook to override ones in `defaults/main.yml`
+
+```yaml
+- hosts: tor-servers
+  roles:
+    - nusenu.relayor
+  vars:
+    tor_ContactInfo: "contact [AT] domain.com"
+    tor_nicknamefile: files/nicknames.csv
+    tor_AbuseEmailAddress: abuse@domain.com
+    tor_package_state: latest
+    tor_signingkeylifetime_days: 30
+```
+
```

Comment:
@lunarthegrey Could you also include the precise command to run?

Reply:
I guess there's

    ansible-playbook yourplaybook.yml -l newserver --tags openbsd

from `README.md`, but maybe ^^that^^ command could be added to a **Quickstart** section you could add toward the top, near your Example Playbook?